opennms-services/src/main/resources/META-INF/opennms/applicationContext-pollerd.xml

```diff
@@ -324,8 +337,39 @@
 		ref="webMonitor">
 		<onmsgi:service-properties>
 			<entry key="implementation"
-				value="org.opennms.netmgt.poller.monitors.WebMonitor" />
+				value="org.opennms.netmgt.poller.ServiceMonitorCamelImpl" />
 		</onmsgi:service-properties>
 	</onmsgi:service>
+	
+	<!-- Active MQ -->
+    <bean id="activemq" class="org.apache.activemq.camel.component.ActiveMQComponent">
+    <property name="brokerURL" value="tcp://localhost:61616" />
+    </bean>
+    
+    <bean id="poller" class="org.opennms.netmgt.poller.remote.Poller"/>
+    
+    <camelContext id="pollerCamelContext" xmlns="http://camel.apache.org/schema/spring">
+    
+        <!-- Bootstrap timer -->
+    <route id="startAvailabilityMonitorPollerTimer">
```

Comment:
This timer is not necessary. The polls will be scheduled by the org.opennms.netmgt.scheduler.LegacyScheduler within the Poller class. That happens outside of this Camel context.